workspace_test.go

```diff
@@ -376,6 +383,7 @@ func TestWorkspacesUpdateByID(t *testing.T) {
 			assert.Equal(t, *options.AutoApply, item.AutoApply)
 			assert.Equal(t, *options.FileTriggersEnabled, item.FileTriggersEnabled)
 			assert.Equal(t, *options.QueueAllRuns, item.QueueAllRuns)
+			assert.Equal(t, *options.Operations, item.Operations)
```

Comment:
In order to keep things (more or less) consistent, this should be moved above `QueueAllRuns` everywhere in this file 😏

Reply:
Oops! Clearly I can't alphabet. Going to make the changes soon. I'm also in the process of working with a developer regarding the CI builds.